sdk/push-fcm/src/main/java/com/klaviyo/pushFcm/KlaviyoRemoteMessage.kt

```diff
@@ -135,6 +136,32 @@ object KlaviyoRemoteMessage {
     val RemoteMessage.notificationCount: Int
         get() = this.data[KlaviyoNotification.NOTIFICATION_COUNT_KEY]?.toInt() ?: 1
 
+    /**
+     * Determine if the message is bearing key-value pairs
+     */
+    val RemoteMessage.hasKlaviyoKeyValuePairs: Boolean
+        get() = this.isKlaviyoMessage && (this.data[KlaviyoNotification.KEY_VALUE_PAIRS_KEY]?.let { true } ?: false)
+
+    /**
+     * Parse out the key-value pairs into a string:string map
+     */
+    val RemoteMessage.keyValuePairs: Map<String,String>?
```

Comment:
Looks like we have a gap in our test coverage, this file has no corresponding test file. Would you mind adding a file in the same dir as `KlaviyoPushServiceTest` to cover your changes, and we'll backfill tests for other stuff in the future.